runtime-decompiler/src/main/java/org/jrd/frontend/frame/overwrite/OverwriteClassDialog.java

```diff
@@ -490,6 +492,21 @@ public void actionPerformed(ActionEvent actionEvent) {
         return compiler;
     }
 
+    public static ClasspathlessCompiler getClasspathlessCompiler(DecompilerWrapper wrapper, boolean hasCompiler) {
+        ClasspathlessCompiler rc;
+        if (hasCompiler) {
+            rc = new RuntimeCompilerConnector.ForeignCompilerWrapper(wrapper);
+        } else {
+            boolean useHostClasses = Config.getConfig().doUseHostSystemClasses();
+            ClasspathlessCompiler.Arguments arguments = new ClasspathlessCompiler
+                    .Arguments()
+                    .useHostSystemClasses(useHostClasses);
```

Comment:
Smuggling changes I see, hostJavaClasses magically extracted to hostSystemClasses I notice.

Reply:
Sorry! Nice reveng the https://github.com/pmikova/java-runtime-decompiler/pull/146 . TY!-)